### PR TITLE
{tools}[intel/2015a] pycosat 0.6.0 + networkx 1.10 w/ Python 2.7.10

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/p/pycosat/pycosat-0.6.0-intel-2015a-Python-2.7.10.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+python = 'Python'
+pyver = '2.7.10'
+versionsuffix = '-%s-%s' % (python, pyver)
+
+dependencies = [(python, pyver)]
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/pycosat/pycosat-0.6.0-intel-2015a-Python-2.7.10.eb]: Make consistent with the other easyconfig.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f2a352f7efcc74bc3e6a for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**